code-style/git/README.md

```diff
@@ -209,6 +209,9 @@ Every place you work will have a different Git flow. At Sparkbox the Git flow is
 
     -  Protip: Sometimes we create pull requests early as a place to collaborate on a solution. Consider labeling these as `DO NOT MERGE` or `WIP`.
 
+
+1. To make the pull request process more consistant, your project may include a [pull request template][]. An [example PR template](../PULL_REQUEST_TEMPLATE.md) is included in the Standard as a starting point.
```

Comment:
be consistent with markdown links... move `.../PULL_REQUEST_TEMPLATE.md` down to the bottom via id.